src/common/reduction/matching/GapMatcher.h

```diff
@@ -36,6 +40,30 @@ class GapMatcher : public AbstractMatcher {
   /// \brief print configuration of GapMatcher
   std::string config(const std::string &prepend) const override;
 
+  void reset_stats() {
```

Comment:
better to use memset , then you don't need to remember adding anything here if you 
extend stats
memset(Stats, 0, sizeof(Stats))